### PR TITLE
doc(changelog) Add missing entries

### PR DESCRIPTION
**Pull request by Hywan**
# Description

This patch adds missing entries in the CHANGELOG for the ongoing release.

# Review

- [ ] ~Add a short description of the the change to the CHANGELOG.md file~ otherwise I've an infinite loop

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,18 +8,26 @@
 ## **[Unreleased]**
 
 ### Added
+- [#2154](https://github.com/wasmerio/wasmer/pull/2154) Implement reference types in the LLVM compiler.
```
Comment (nlewycky) [inline CHANGELOG.md:11]: This PR was a branch-to-branch merge, it didn't go to master.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,18 +8,26 @@
...
+- [#2103](https://github.com/wasmerio/wasmer/pull/2103) Add middleware (incl. metering) in the C API.
+- [#2003](https://github.com/wasmerio/wasmer/pull/2003) Wasmer works with musl, and is built, tested and packaged for musl.
+- [#2116](https://github.com/wasmerio/wasmer/pull/2116) Add a package for Windows that is not an installed, but all the `lib` and `include` files as for macOS and Linux.
```
Comment (Hywan) [inline CHANGELOG.md:14]:
```suggestion
- [#2116](https://github.com/wasmerio/wasmer/pull/2116) Add a package for Windows that is not an installer, but all the `lib` and `include` files as for macOS and Linux.
```

File: CHANGELOG.md (hunk 1 above)
Comment (Hywan) [inline CHANGELOG.md:11]:
```suggestion
```

**Review by syrusakbary — APPROVED** [review]

**Comment by syrusakbary** [comment]
Merging manually

**Pull request merged**